### PR TITLE
Upgrade unity Version + Hand Aim

### PR DESCRIPTION
**Pull request by alvmivan**

### CODE REVIEW

**Comment by alvmivan** [comment]
[[Unity] Update Unity project to last LTS](https://trello.com/c/zkB4m7fB/19-unity-update-unity-project-to-last-lts)

File: Gamework Game/Assets/Scripts/Player/PlayerAnim.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+namespace Player
+{
+    
+    
+    
+    public class PlayerAnim : MonoBehaviour
+    {
+        [SerializeField] private PlayerPhysics playerPhysics;
+        private IPlayerPhysics physics;
+        public Animator faceAnimation;
```
Comment (ManuBenito259) [inline Gamework Game/Assets/Scripts/Player/PlayerAnim.cs]: Stares in Sergio

File: Gamework Game/Assets/Scripts/Player/PlayerArm.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+    public void AimTo(Vector2 point)
+    {
+        realDirection = (Vector3) point - transform.position;
+    }
+    
+    private void FixedUpdate()
+    {
+        const float armDampRotation = 50f;
```
Comment (ManuBenito259) [inline Gamework Game/Assets/Scripts/Player/PlayerArm.cs:32]: Maybe it would be usefull to expose it in case we need twitching the value

Comment (alvmivan) [reply Gamework Game/Assets/Scripts/Player/PlayerArm.cs:32]: as long as it's placeholder, it's ok

File: Gamework Game/Assets/Scripts/Player/PlayerAnim.cs (hunk 1 above)
Comment (alvmivan) [reply Gamework Game/Assets/Scripts/Player/PlayerAnim.cs]: done

**Pull request merged**